### PR TITLE
Rancher CLI not compatible with feature charts

### PR DESCRIPTION
**Pull request by btat**
Closes https://github.com/rancher/docs/issues/4044

### CODE REVIEW

**Review by jtravee — APPROVED** [review]

**Review by anupama2501 — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 1]
```diff
@@ -20,25 +20,25 @@ Starting in Rancher v2.6.0, a new versioning scheme for Rancher feature charts w
...
 | rancher-logging | 100.0.0+up3.12.0 | 100.0.1+up3.15.0 |
-| rancher-longhorn | 100.0.0+up1.1.2 | 100.1.1+up1.2.3 |                                 
+| rancher-longhorn | 100.0.0+up1.1.2 | 100.1.1+up1.2.3 |
 | rancher-monitoring | 100.0.0+up16.6.0 | 100.1.0+up19.0.3
-| rancher-sriov (experimental) | 100.0.0+up0.1.0 | 100.0.1+up0.1.0 |                                
+| rancher-sriov (experimental) | 100.0.0+up0.1.0 | 100.0.1+up0.1.0 |
 | rancher-vsphere-cpi | 100.0.0 | 100.1.0+up1.0.100
 | rancher-vsphere-csi | 100.0.0 | 100.1.0+up2.3.0 |
-| rancher-wins-upgrader | 100.0.0+up0.0.1 | 100.0.0+up0.0.1 |                                                   
+| rancher-wins-upgrader | 100.0.0+up0.0.1 | 100.0.0+up0.0.1 |
```
[the remark this answers is not in the document]
Comment (btat) [reply content/rancher/v2.6/en/helm-charts/_index.md:36]: Add Neuvector and updated versions to the table

[the remark this answers is not in the document]
Comment (anupama2501) [reply content/rancher/v2.6/en/helm-charts/_index.md:36]: Following are the supported min versions of apps supported on rancher.
`rancher-alerting-drivers`: `100.0.2`
`rancher-backup`: `2.0.1`
`rancher-cis-benchmark`: `2.0.1`
`rancher-gatekeeper`: `100.0.1+up3.6.0`
`rancher-vsphere-cpi` no min version, only version available `100.3.0+up1.2.1`
`rancher-vsphere-csi` no min version, only version available `100.3.0+up2.5.1-rancher1`
`rancher-wins-upgrader` : `0.0.100`

[the remark this answers is not in the document]
Comment (btat) [reply content/rancher/v2.6/en/helm-charts/_index.md:36]: Minimum versions updated

**Review by anupama2501 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**